### DISCUSSION TITLE
AOC23 - Advent of Code 2023 implemented in Arturo

### DISCUSSION THREAD

**Discussion by RickBarretto** [arturo-lang/arturo#1297] [General]
The idea of this Thread is to discuss about the Advent of Code 2023 implemented in Arturo.

## Event
**Challenges**: [AOC23](https://adventofcode.com/)
**Date:** 1st, December » 25th, December
**Hour:** 12am, UTC-5

## Discord
We've created the same thread on Discord: [Thread: AOC 2023 - Advent of Code 2023 in Arturo][discord]

[discord]: https://discord.com/channels/765519132186640445/1171429149353726062

## Recommendations
The project should use the tag **_#aoc2023_** and **_#arturo_**, 
as you can see in the image bellow.

![image](https://github.com/arturo-lang/arturo/assets/78623871/9a91f3ad-74b3-443b-a7bc-6c8c70562122)

## Showcase
See other implementations on the [Index of Repositories][index].

---

Show us your own implementations on this Thread and have fun! 🙂🎉

[index]: https://github.com/search?q=topic%3Aarturo+topic%3Aaoc2023&type=repositories

**Comment by Ryton** [comment]
Count me in!  💯 

_Disclaimer:_  
Expect horribly unclear and inefficient code, that might work for a minority of cases! ;-)

**Reply by drkameleon** [reply]
Hahaha! Awesome!!! 🚀

**Reply by RickBarretto** [reply]
Hi, @Ryton! Remembering that the AOC23 is near to begin! 🎉🎉🎉

**Comment by RickBarretto** [comment]
I created [my very first repository][repo] if you guys want to use as reference: 

Also, it's possible to find other implementations using [the query `topic:arturo topic:aoc2023`][query]

[repo]: https://github.com/RickBarretto/aoc-arturo
[query]: https://github.com/search?q=topic%3Aarturo+topic%3Aaoc2023&type=repositories
